app/views/dashboard/callouts/edit.html.haml

```diff
@@ -0,0 +1,3 @@
+.col-12.col-md-8
+  %h5.card-title Edit callout
+  = render partial: 'form', locals: { callout: @callout, url: dashboard_callout_path(@callout) }
```

Comment:
I don't think you need to pass url because Rails will generate a correct base on state of model

you should just password `simple_form_for [:dashboard, callout]`